docs/guides/deployment/kubernetes.md

```diff
@@ -69,15 +76,118 @@ Besides a cluster, the other critical resource we'll need is a container registr
+Now let's switch over to Prefect Cloud, where we'll create a new work pool. 
+Hit the plus button on the work pools page and choose Kubernetes from the list of options. 
+On the next page, set Finished Job TTL to 60 so that completed flow runs are cleaned up, and set Pod Watch Timeout Seconds to 300, especially if you are using a __serverless__ type node pool since these tend to have longer startup times. 
+You may also want to set a custom namespace, such as `prefect`. 
+Generally you should leave the cluster config blank as the worker will already be provisioned with appropriate access and permissions. 
+Finally, leave the image field blank as we'll override that in each deployment.
 
 ## Deploy a worker using Helm
 
-With our cluster and work pool created, it's time to deploy a worker, which will take our flows and run them as Kubernetes jobs. The best way to deploy a worker is using Helm. Follow [this guide](/guides/deployment/helm-worker/) and then come back here.
+With our cluster and work pool created, it's time to deploy a worker, which will take our flows and run them as Kubernetes jobs. 
+The best way to deploy a worker is using the [Prefect Helm Chart](https://github.com/PrefectHQ/prefect-helm/tree/main/charts/prefect-worker). 
+
+### Step 1: Add Prefect Helm repository
+
+Add the Prefect Helm repository to your Helm client:
+
+<div class="terminal">
+```bash
+helm repo add prefect https://prefecthq.github.io/prefect-helm
+helm repo update
+```
+</div>
+
+### Step 2: Create a namespace for Prefect worker
+
+Create a new namespace in your Kubernetes cluster to deploy the Prefect worker:
+
+<div class="terminal">
+```bash
+kubectl create namespace prefect
+```
+</div>
+
+### Step 3: Create a Kubernetes secret for the Prefect API key
+
+Create a file named `api-key.yaml` with the following contents:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: prefect-api-key
+  namespace: prefect
+type: Opaque
+data:
+  key:  <base64-encoded-api-key>
+```
+
+Replace `<base64-encoded-api-key>` with your Prefect Cloud API key encoded in base64. 
+The helm chart looks for a secret of this name and schema, this can be overridden in the `values.yaml`.
+
+You can use the following command to generate the base64-encoded value:
+
+<div class="terminal">
+```bash
+echo -n "your-prefect-cloud-api-key" | base64
+```
+</div>
+
+Apply the `api-key.yaml` file to create the Kubernetes secret:
+
+<div class="terminal">
+```bash
+kubectl apply -f api-key.yaml
+```
+</div>
```

Comment:
This can also be done in a one liner:
```bash
kubectl create secret generic prefect-api-key --from-literal=key=your-prefect-cloud-api-key
```
Though @gabcoyne knows better than me which is best practice.

Reply:
Great! I'd love to add the one liner for brevity. I expect George would agree, and will give a few days to weigh in. Otherwise, let's cut it down.

Reply:
The one-liner is fine, but there are two caveats that users should be aware of:

1. The key may be logged in your terminal history (e.g. `bash_history` or `history` command); you can usually avoid this by adding a space before the command when you run it
2. Programs (including command-line parameters) are visible to other users on your system, so this can be an issue on multi-user systems

In practice, these are not big risks and we all do stuff like this. 🤷

Reply:
updated to 1 line